site/content/in-dev/unreleased/evolution.md

```diff
@@ -0,0 +1,115 @@
+
+Each Polaris release may support multiple Persistence [implementations](../metastores),
+for example, "EclipseLink" (deprecated) and "JDBC" (current).
+
+Each type of Persistence evolves individually. Within each Persistence type, Polaris
+attempts to support rolling upgrades (both version X and X + 1 servers running at the
+same time).
+
+However, migrating between different Persistence types is not supported in a rolling
+upgrade manner (for example, migrating from "EclipseLink" to "JDBC"). Polaris provides
+[tools](https://github.com/apache/polaris-tools/) for migrating between different
+catalogs and those tools may be used to migrate between different Persistence types
+as well. Service interruption (downtime) should be expected in those cases.
+
+## Using Polaris as a Build-Time Dependency
+
+Polaris produces several jars. These jars or custom builds of Polaris code may be used in
+downstream projects according to the terms of the license included into Polaris distributions.
+
+The minimal version of the JRE required by Polaris code (compilation target) may be updated in
+any release. Different Polaris jars may have different minimal JRE version requirements.
+
+Changes in Java class should be expected at any time regardless of the module name or
+whether the class / method is `public` or `private`.
+
+This approach is not meant to discourage the use of Polaris code in downstream projects, but
+to allow more flexibility in evolving the codebase to support new catalog-level features
+and improve code efficiency. Maintainers of downstream projects are encouraged to join Polaris 
+mailing lists to monitor project changes, suggest improvements, and engage with the Polaris
+community in case of specific compatibility concerns.   
+
+## Semantic Versioning
+
+Polaris strives to follow [Semantic Versioning](https://semver.org/) conventions both with
+respect REST APIs (beta and experimental APIs excepted), [Polaris Policies](../policy/)
+and user-facing [configuration](../configuration/).
+
+The following are some examples of Polaris approach to SemVer in REST APIs / configuration.
+These examples are for illustration purposes and should not be considered to be
+exhaustive.
+
+Polaris implementing an optional Iceberg REST Catalog feature that was unimplemented
+in the previous release is not considered a major change.
+
+Supporting a new revision of the Iceberg REST Catalog spec in a backward-compatible way
+is not considered a major change. Specifically, supporting new REST API prefixes (e.g. `v2`)
+is not a mojor change because it does not affect older clients.
+
+Changing the implementation of an Iceberg REST Catalog feature / endpoint in a non-backward
+compatible way (e.g. removing or renaming a request parameter) is a major change.
+
+Dropping support for a configuration property with the `polaris.` name prefix is a major change.
+
+Dropping support for any previously defined [Policy](../policy/) type or property is a major change.
```

Comment:
Based on the format, it might be good to make this a `<ul>`

Reply:
good point - updated